hbase-rsgroup/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupsKillRS.java

```diff
@@ -268,7 +269,9 @@ public void testLowerMetaGroupVersion() throws Exception{
     assertTrue(VersionInfo.compareVersion(originVersion,
         master.getRegionServerVersion(getServerName(servers.iterator().next()))) > 0);
     LOG.debug("wait for META assigned...");
-    TEST_UTIL.waitTableAvailable(tableName, 30000);
+    TEST_UTIL.waitFor(60000, () -> !TEST_UTIL.getHBaseCluster().getMaster().getProcedures().stream()
```

Comment:
So this is the root cause.

Reply:
Yes. This is the root cause. In my test, the waitUntilAllRegionsAssigned and waitTableAvailable not worked for meta table...